Modules/Scripted/QReads/QReads.py

```diff
@@ -741,6 +750,11 @@ def loadDICOMDataDirectory(dicomDataDir):
 
           loadedNodeIDs.append(nodeID)
 
```

Comment:
Would be good to add a try/except around this so that the dialog doesn't get wedged in an open state on error.  Instead a different message box could display that there was an error.

Reply:
Agreed. A context manager would allow to nicely handle this.